### PR TITLE
Remove one-time EZProxy announcement email

### PR DESCRIPTION
**Pull request by sahilgrewal8072**
[//]: # (Thank you for uploading a PR to the Wikipedia Library!)

## Description
Modified and deleted 8 files as requested in task (T275041).Out of the 8 files, 4 are modified and 4 are deleted succesfully.

## Rationale
[//]: # The code changes will remove one-time EZProxy announcement email.

## Phabricator Ticket
[//]: # https://phabricator.wikimedia.org/T275041

## How Has This Been Tested?
[//]: # All the changes are Cross verified and tested.

### CODE REVIEW

File: TWLight/emails/tasks.py [hunk 1]
```diff
@@ -39,7 +39,6 @@
 from TWLight.emails.signals import ContactUs
 from TWLight.resources.models import AccessCode, Partner
 from TWLight.users.groups import get_restricted
-from TWLight.users.signals import Notice, ProjectPage2021Launch, ProxyBundleLaunch
```
Comment (Samwalton9) [inline TWLight/emails/tasks.py:42]: `Notice` and `ProjectPage2021Launch` are used for other signals, so you only need to remove `, ProxyBundleLaunch` here.

**Comment by jsnshrmn** [comment]
I renamed this PR to match the phab task so it's easier for us to understand our outstanding PRs at a glance.

**Comment by Samwalton9** [comment]
Sorry we didn't review this yet :) Why did you close this and open https://github.com/WikipediaLibrary/TWLight/pull/646 ? Is there any difference in the code changes?

**Comment by sahilgrewal8072** [comment]
Sorry, Sam I deleted it by mistake.

**Pull request closed without merging**